src/unittests/sr5.NetworkDevices.spec.ts

```diff
@@ -73,19 +73,33 @@ export const shadowrunNetworkDevices = (context: QuenchBatchContext) => {
             assert.strictEqual(token?.id, resolvedToken?.id);
         });
 
-        it('connect a device controller to a network device', async () => {
+        it('connect a device controller to an item network device', async () => {
             const controller = await testItem.create({type: 'device'});
             const device = await testItem.create({type: 'weapon'});
 
             await NetworkDeviceFlow.addDeviceToNetwork(controller, device);
 
             assert.strictEqual(device.system.technology.networkController, controller.uuid);
+            assert.strictEqual(device.getNetworkController(), controller.uuid);
             assert.strictEqual(await NetworkDeviceFlow.resolveLink(device.system.technology.networkController), controller);
 
             assert.deepEqual(controller.system.networkDevices, [device.uuid]);
         });
 
-        it('connect a host controller to a network device', async () => {
+        it('connect a device controller to a vehicle network device', async () => {
```

Comment:
You're amazing for adding tests :)

Reply:
Thank you! You're amazing for being such an active maintainer! Makes it easy to want to contribute 😃